packages/custom_lint_builder/test/analyzer_converter_test.dart

```diff
@@ -83,4 +83,72 @@ void main() {
       },
     );
   });
+
+  test('Respects configSeverities when converting errors', () {
```

Comment:
Isn't this covered by the previous tests? I'd rather not have tests that are so tied to the implementation details.

Reply:
You're right. I will remove this test because it isn't needed here.